__tests__/articles.test.js

```diff
@@ -0,0 +1,87 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/articles/:article_id", () => {
+  describe("GET", () => {
+    describe("Status 200", () => {
+      it("should return the correct article given a valid article_id", () => {
+        return request(app)
+          .get("/api/articles/1")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.article).toHaveProperty("author");
+            expect(body.article).toHaveProperty("title");
+          });
+      });
+
+      it("should return an article with the correct structure", () => {
+        return request(app)
+          .get("/api/articles/1")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.article).toMatchObject({
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
```

Comment:
I enjoy using `expect.any` when we want our tests to be flexible, however, when you are making a request for article_id `1`... would we expect the values to be any string or would we expect the values to be exactly what they should be based on whats inside the test data for article_id `1` ? 👀 
Do we want to be flexible here, or rigid ?